AOP_Strip_Processor_Advanced_Ortho Product.md

```diff
@@ -0,0 +1,201 @@
+
+* Enable/disable output tiling. Choices are 'true' or 'false'. 
+    * Default = "False".  If 'true the 'ortho_tiling_scheme' input must be set.
+    * Required = false
+    * type": "string"
+    * Name = "enable_tiling"
+* Comma-separated list of numeric strip parts to process. 
+    * Default = process all strip parts
+    * Required = false,
+    * Type = "string",
+    * Name = "parts"
+* Ortho EPSG projection. String 'UTM' is also allowed. Specify along with ortho_pixel_size. Overridden by ortho_tiling_scheme if specified. 
+    * Default = 'EPSG:4326'  
+    * Required = false
+    * type = "string"
+    * name = "ortho_epsg"
+* Ortho pixel size in meters. Specify along with ortho_epsg. Overridden by ortho_tiling_scheme if specified. 
+    * Default = 'Auto', which uses the pixel size of the input data.",
+    * Required = false,
+    * type = "string"
+    * Name = "ortho_pixel_size"
+* Ortho tiling scheme and zoom level, e.g. 'DGHalfMeter:18'. Overrides ortho_epsg and ortho_pixel_size.
+    * Required = false,
+    * type = "string"
+    * name  "ortho_tiling_scheme"
+* Ortho DEM specifier. Options are 'NED', 'SRTM30', 'SRTM90'. 
+    * Default = 'SRTM90'
+    * Required = false
+    * Type = "string"
+    * Name = "ortho_dem_specifier"
+* Ortho pixel interpolation type. Options are 'Nearest', 'Bilinear', 'Cubic'. 
+    * Default is 'Cubic’
+    * Required": false,
+    * type = "string",
+    * name = "ortho_interpolation_type"
+* AComp AOD grid size in meters per pixel.
+    * Default is 10.
+    * Required = false,
+    * type = "string",
+    * name = "acomp_aod_grid_size"
+* Dynamic range adjustment mode. Options are 'IntensityAdjust', 'BaseLayerMatch'. 
+    * Default is 'IntensityAdjust' 
+        * 'BaseLayerMatch' is currently only compatible with EPSG:4326.",
+    * Required = false,
+    * type = "string",
+    * name = "dra_mode"
+* Dynamic range adjustment low cutoff percentage. Range is from 0 to 100. 
+    * Default is 0.5. Only used for 'IntensityAdjust' mode.","
+    * Required": false,
+    * type = "string"
+    * Name = "dra_low_cutoff"
+* Dynamic range adjustment high cutoff percentage. Range is from 0 to 100. 
+    * Default is 99.95. Only used for 'IntensityAdjust' mode
+    * Required = false
+    * type = "string"
+    * name = "dra_high_cutoff"
+* Dynamic range adjustment gamma value. 
+    * Default = 1.25. Only used for 'IntensityAdjust' mode.
+    * Required = false
+    * type = "string"
+    * name = "dra_gamma"
+* Dynamic range adjustment output bit depth. Choices are 8 or 16. 
+    * Default = 8. Only used for 'IntensityAdjust' mode.
+    * Required = false
+    * type = "string"
+    * name = "dra_bit_depth"
+* Zoom level (i.e. size) of output tiles if tiling is enabled. 
+    * Default = 12.
+    * Required = false
+    * type = "string",
+    * name = "tiling_zoom_level"
+
+**_Sample Output: Test Run 4/14/16, On R&D Cluster_**
```

Comment:
This sample output from ipython could use some better formating, at least fewer spaces